inlong-dashboard/src/components/MetaData/StorageGreenplum.tsx

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
Suggest changing the file name to `SinkGreenplum`, because the backend naming is `XxxSink`.

Reply:
Okay.